docs_user/modules/proc_deploying-the-bare-metal-provisioning-service.adoc

```diff
@@ -1,45 +1,42 @@
 [id="deploying-the-bare-metal-provisioning-service_{context}"]
 
-= Deploying the Bare Metal Provisioning service
+= Deploying the {bare_metal}
 
-Applying the configuration to deploy the {bare_metal_first_ref}.
+To deploy the {bare_metal_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR) that has the {bare_metal} disabled. The operator applies the configuration and starts the Bare Metal Provisioning services. After the services are running, the {bare_metal} automatically begins polling the power state of the bare metal nodes that it manages.
 
 [NOTE]
-By default, newer versions of the {bare_metal} contain a more restrictive access control model while also becoming multi-tenant aware. As a result,  bare metal nodes might be missing from a `openstack baremetal node list` command after upgrade. Your nodes have not been deleted, but you must set the `owner` field on each bare metal node due to the increased access restrictions in the role-based access control (RBAC) model. Because this involves access controls and the model of use which can be site specific, it is highly recommended that you identify the "project" to "own" the bare metal nodes.
-
-//kgillga: Does "after upgrade" mean after upgrading to RHOSP 17.1 or after adopting RHOSO 18.0?
+By default, newer versions of the {bare_metal} contain a more restrictive access control model while also becoming multi-tenant aware. As a result, bare metal nodes might be missing from a `openstack baremetal node list` command after you upgrade. Your nodes are not deleted. You must set the `owner` field on each bare metal node due to the increased access restrictions in the role-based access control (RBAC) model. Because this involves access controls and the model of use can be site specific, you should identify which project owns the bare metal nodes.
```

Comment:
What does "multi-tenant aware" mean? Does it mean that newer versions of bare metal support multi tenancy?